adodbapi/quick_reference.md

```diff
@@ -59,11 +59,11 @@ Windows. Note that you are not permitted load the 32 bit "ACE" provider
 if you have any 64-bit Office components installed. Conventional wisdom
 says that you must use 64 bit Python in this case. However, see the
 answer in
-<http://stackoverflow.com/questions/12270453/ms-access-db-engine-32-bit-with-office-64-bit>.
+<https://stackoverflow.com/questions/12270453/ms-access-db-engine-32-bit-with-office-64-bit>.
 If you decide to try hacking the installers, you may find
-<http://www.pantaray.com/msi_super_orca.html> to be a useful alternative
+<https://www.pantaray.com/msi_super_orca.html> to be a useful alternative
 to Orca. My experience is that such a [hacked installer (like this
-one)](http://shares.digvil.info/redis) can also be used on machines
+one)](http://shares.digvil.info/redis) (dead link) can also be used on machines
```

Comment:
This is a completely dead link that was never archived. The only referenec to it I could found was this mailing list post by @vernondcole https://python-win32.python.narkive.com/3GdgqjEL/opening-access-mdb-database-adodb-connection-der-provider-kann-nicht-gefunden-werden#post3

Would it be better to just remove this sentence entirely ?

Reply:
yeah, pls remove it!